raiden/tests/scenarios/bf2_long_running.yaml

```diff
@@ -0,0 +1,335 @@
+          tasks:
+            - wait: 3
+      - parallel:
+          name: "Sanity check"
+          tasks:
+            - assert: {from: 0, to: 1, total_deposit: 100, balance: 40, state: "opened"}
+            - assert: {from: 1, to: 0, total_deposit: 100, balance: 160, state: "opened"}
+            - assert: {from: 1, to: 2, total_deposit: 100, balance: 200, state: "opened"}
+            - assert: {from: 2, to: 1, total_deposit: 100, balance: 0, state: "opened"}
+            - assert: {from: 2, to: 4, total_deposit: 150, balance: 90, state: "opened"}
+            - assert: {from: 4, to: 2, total_deposit: 0, balance: 60, state: "opened"}
+            - assert: {from: 4, to: 0, total_deposit: 25, balance: 15, state: "opened"}
+            - assert: {from: 0, to: 4, total_deposit: 10, balance: 20, state: "opened"}
+            - assert: {from: 2, to: 3, total_deposit: 101, balance: 101, state: "opened"}
+            - assert: {from: 3, to: 2, total_deposit: 101, balance: 101, state: "opened"}
+      - serial:
+          name:
+          tasks:
+            # stop node B
+            - stop_node: 1
+            # give it time to shutdown
+            - wait: 10
+      - serial:
+          name: "Multi hop payment 2"
+          tasks:
+            # A sends 10 tokens to C (using the A <-> E <-> C route)
+            - transfer: {from: 0, to: 2, amount: 10, lock_timeout: 30}
+            - wait: 5
+      - parallel:
+          name: "Assert transfer was executed correctly"
+          tasks:
+            - assert: {from: 2, to: 1, total_deposit: 100, balance: 0, state: "opened"}
+            - assert: {from: 0, to: 1, total_deposit: 100, balance: 40, state: "opened"}
+            - assert: {from: 4, to: 0, total_deposit: 25, balance: 25, state: "opened"}
+            - assert: {from: 0, to: 4, total_deposit: 10, balance: 10, state: "opened"}
+            - assert: {from: 2, to: 4, total_deposit: 150, balance: 100, state: "opened"}
+            - assert: {from: 4, to: 2, total_deposit: 0, balance: 50, state: "opened"}
+            - assert: {from: 0, to: 1, total_deposit: 100, balance: 40, state: "opened"}
+      - parallel:
+          name: "Check edge cases again"
+          tasks:
+            # try to open a channel with an offline node, but the channel already exists
+            - open_channel: {from: 0, to: 1, total_deposit: 200, expected_http_status: 409}
+            # Try to make a payment to an offline node
+            - transfer: {from: 0, to: 1, amount: 1, expected_http_status: 409, lock_timeout: 30}
+      - serial:
+          name: "Restarting node B"
+          tasks:
+            - start_node: 1
+      - parallel:
+          name: "Checking for the D <-> C channel state"
+          tasks:
+            - assert: {from: 2, to: 3, total_deposit: 101, balance: 101, state: "opened"}
+            - assert: {from: 3, to: 2, total_deposit: 101, balance: 101, state: "opened"}
+              #      - serial:
```

Comment:
If you don't want this part in the scenario, then please remove it instead of leaving it as a comment.